README.md

```diff
@@ -22,7 +22,7 @@ docker run -v $(pwd):/root/devilution -e MAKE_BUILD=pdb diasurgical/riivaaja
 
 ## Building a new image
 
-Place the files for Visual C++ 5 SP3 (DevStudio_5.10) and Visual C++ SP5 PP5 (VS6) in the Riivaaja root folder. Make sure the versions of VC actually support optimizations (Professional and higher).
+Place the files for Visual C++ 5 SP3 (DevStudio_5.10) and Visual C++ SP5 PP5 (VS6) in the Riivaaja root folder. Make sure the versions of VC actually support optimizations (Professional and higher). Use Devilution-comparer to generate a orig_full.asm and put it in the root.
```

Comment:
Could you add that this is optional and only needed if you want to get the comparation results?